Gemfile

```
@@ -65,7 +65,7 @@ if ENV['NO_ACCEPTANCE'] != 'true'
       # use the pinned version
       gem 'beaker', '~> 4.1'
     end
-    gem 'beaker-hostgenerator', '~> 2.2.3'
+    gem 'beaker-hostgenerator', '~> 2.4.0'
```

This specificity is just going to lead to issues later. I suggest:
```suggestion
    gem 'beaker-hostgenerator', '~> 2.4'
```
Which will allow any `2.x` gem after 2.4

---

My bad, didn't realize y'all would rather float on the `y`.

---

no worries! not sure how it got in that state to begin with, easy enough fix up :)

---

Updated the code to float the minor version